Add community projects page (#829)
tigerbeetle/tigerbeetle: docs/COMMUNITY_PROJECTS.md

# Community Projects

> This page celebrates the incredible work done by community members. 😊

Reviewer: ```suggestion
Documenting the incredible work done by community members. 😊
```

To also remove one of the "this page" phrases, which occurs again in the second paragraph.

If you've made a project and would like to be featured here, open a
pull request for this page!

## Applications in the Wild

* [Plain Text Accounting with TigerBeetle](https://github.com/pondersource/pta-tb-experiment#pta-tb-experiment)

> ## Database Clients Made With ❤️  By the Community

Reviewer: ```suggestion
## Database Clients Made With ❤️ By the Community
```

Remove double space.

They are not officially supported by TigerBeetle but we do our best to

> support them anyway!

Reviewer: Third-party clients not yet officially supported by TigerBeetle.

Author: Done!

> Keep in mind before our first release there will be frequent breaking

Reviewer: ```suggestion
Keep in mind that before our first release there will be frequent breaking
```

changes. We integration-test all first-party clients but third-party
clients may lag behind.

> * [tigerbeetlex](https://github.com/rbino/tigerbeetlex): An Elixir client for TigerBeetle
> * [tigerbeetle-cpp](https://github.com/kassane/tigerbeetle-cpp): TigerBeetle C++ Client
> * [tigerbeetle-rs](https://github.com/ZetaNumbers/tigerbeetle-rs): Safe bindings to TigerBeetle

Reviewer: ```suggestion
* [tigerbeetlex](https://github.com/rbino/tigerbeetlex): An Elixir client
* [tigerbeetle-cpp](https://github.com/kassane/tigerbeetle-cpp): A C++ client
* [tigerbeetle-rs](https://github.com/ZetaNumbers/tigerbeetle-rs): A Rust client
```

Need a client for Go, Java, Node.js, or .NET? We've got you covered
with [first-party
clients](https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients).

## Other

> * [A Jepsen Test for TigerBeetle](https://github.com/nurturenature/jepsen-tigerbeetle)

Reviewer: This could (with 50/50 probability) be moved under Applications in the Wild.

Author: Done!
